src/tic_tac_toe/setup.clj

```diff
@@ -1,70 +1,59 @@
 (ns tic-tac-toe.setup
   (:use [clojure.java.io :as fs]
-        [tic-tac-toe.menu-selector :only [get-selection]]
-        [tic-tac-toe.menu-messages :only [messages]]
-        [tic-tac-toe.save-exit :only [save? save-and-exit]])
-  (:require [tic-tac-toe.board :as board]
-            [tic-tac-toe.computer :as computer]
-            [tic-tac-toe.game :as game]
-            [tic-tac-toe.human :as human]
-            [tic-tac-toe.io :as io]
+        [tic-tac-toe.menu.menu-selector :only [get-menu-selection]]
+        [tic-tac-toe.menu.menu-messages :only [messages]]
+        [tic-tac-toe.game-flow.game-saver :only [save? quick-save]]
+        [tic-tac-toe.setup.quick-save :only [quick-save-options]])
+  (:require [tic-tac-toe.setup.new-game :as new-game]
+            [tic-tac-toe.setup.saved-game :as saved-game]
+            [tic-tac-toe.game-flow.game-loop :as loop]
+            [tic-tac-toe.ui.io :as io]
             [tic-tac-toe.read-write.reader :as reader]
-            [tic-tac-toe.user-interface :as ui]
+            [tic-tac-toe.read-write.timestamper :as timestamper]
```

Comment:
I _think_ you may not need the `:as timestamper`. Do you know if that is accurate?

Reply:
You are correct.  I removed that and then had to change `timestamper/date-difference` to `date-difference`.